packages/shared/tailwind.config.js

```diff
@@ -114,6 +114,7 @@ module.exports = {
     boxShadow,
     opacity: {
       0: '0',
+      8: '0.08',
```

Comment:
Oh, nice, didn't know you can use the values here through `/8` from the border implementation.